sphinx/locale/fr/LC_MESSAGES/sphinx.po

```diff
@@ -1,7 +1,7 @@
-# Translations template for Sphinx.
+# French translations for Sphinx.
```

Comment:
Hmm? Did you translate this manually?
We use transifex.com to manage translations.
So you don't change *.po file manually (at least, in master branch).